backend/app/configs/user_verifier/brevo.yaml

```
@@ -0,0 +1,4 @@
+name: brevo
+sender_name: MarketData
+sender_email: stock.market.data.india@gmail.com
+brevo_api_key_name: BREVO_API_KEY
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)